apps/Typescript-ObjC/.gitignore

```diff
@@ -0,0 +1,63 @@
+# OSX
```

Comment:
Is this gitignore based off what we currently do or is it a community-sourced best practice for typescript apps?

Reply:
I just deleted this file in favor of the one in `apps/` that covers *all* sample apps.